src/v2/Components/FlashBanner/index.tsx

```diff
@@ -49,10 +49,15 @@ export const FlashBanner: React.FC<FlashBannerProps> = props => {
 
   let content
 
-  if (contentCode === "email_confirmation_cta") {
-    content = <EmailConfirmationCTA />
-  } else {
-    content = contentMap[contentCode]
+  switch (contentCode) {
+    case "email_confirmation_cta":
+      content = <EmailConfirmationCTA />
+      break
+    case "expired_token":
+      content = <EmailConfirmationLinkExpired />
+      break
+    default:
+      content = contentMap[contentCode]
```

Comment:
Nice 👍 

This is a pattern that I often use to make make this a little less verbose:

```jsx
const getContent = () => {
  switch (contentCode) {
    case "email_confirmation_cta":
      return  <EmailConfirmationCTA />	
    case "expired_token":    
      return <EmailConfirmationLinkExpired />
    default:
      return contentMap[contentCode]
}

const content = getContent() 
```

Reply:
what if you needed, say, a value from a hook? Then one would have to add arguments and pass things around, etc. But that said, its no horible to moveout of function body, just not strictly necessary. Its also nice to read a component top down and see exactly whats going on, versus having to jump all around. This falls apart of course if the component is more complex / longer, but for something like this, keeping it nice and dense increases readability.